### ISSUE TITLE
Error message "Server Errors" related to issue 188 and 186

### ISSUE THREAD

**Issue by clvnong** [mixcore/mix.core#277]
Hi, I have similar issue as found in issue 188 and 186.
I have follow the suggestion given, downloaded the source from develop branch and run with Docker "docker-compose up" command. However, when I tried setup my first web site. I am getting "Server Error" message.

Not sure what I am doing wrong here. Please advice.

Regards,
Calvin Ong

**Comment by Smilefounder** [comment]
Hi @clvnong , Could you share with me some screenshot or the step to reproduce?
I ran 2 steps below and it is fine
```
docker-compose build
docker-compose up
```
Open browser and type to the address: http://192.168.99.100:5000/init (IP should be the same with your docker info).

Note: Please do note that you must remove any "appsettings.json" file in the folder "\src\Mix.Cms.Web" before run docker-compose build.

![image](https://user-images.githubusercontent.com/3785721/95898746-b7e0b580-0db9-11eb-98a7-a9ac800c7ddb.png)

**Comment by clvnong** [comment]
Hi Huy,
Let me try again with removing the appsetting.json.

Regards,
Calvin Ong

**Comment by clvnong** [comment]
Hi Huy,
After following your instructions. I am still getting the same error after
clicking "Connect".
Please refer to the screenshot below.

[image: image.png]

Regards,
Calvin Ong

**Comment by Smilefounder** [comment]
Hi @clvnong, kindly help re attach the image again as I can not see the image via https://github.com/mixcore/mix.core/issues/277

Thanks.

**Comment by Smilefounder** [comment]
Hi @clvnong I'm able to reproduce the issue from my side. Let me double-check and will provide any solution to fix the issue asap.

**Comment by clvnong** [comment]
Hi Huy,
Please find attachment.

**Comment by clvnong** [comment]
<img width="450" alt="Picture 1" src="https://user-images.githubusercontent.com/48722009/95966952-47c84300-0e3e-11eb-8827-d387d9ac55c1.png">

**Comment by Smilefounder** [comment]
Hi @clvnong 
Kindly try with the following solution:
1. Access to phpmyadmin (http://192.168.99.100:8080 username: mixcore, pass: P@ssw0rd)
2. Access mixcore database
3. Select all existing table
4. Drop all DB
5. Try to install again

Let me know if this work.

**Comment by clvnong** [comment]
Hi Huy,
There is no table in the mixcore database.
<img width="451" alt="Picture 2" src="https://user-images.githubusercontent.com/48722009/95985294-02643f80-0e57-11eb-8d3a-b0e3853171ae.png">

**Comment by Smilefounder** [comment]
Hi @clvnong 

If you are using docker, I think you should use mysql from docker component like the below image. As I'm not sure if it is possible to connect from docker network to your localhost or not.

![image](https://user-images.githubusercontent.com/3785721/95990453-bd8bd900-0e55-11eb-8c6f-d69f1ae8d06c.png)

**Comment by clvnong** [comment]
Hi,
You are right! I should connect it with container name instead of localhost.
It is woking now. Thanks.

Regards,
Calvin Ong
